### PR TITLE
Domains: Google Apps: Add loading placeholder

### PR DESCRIPTION
**Pull request by klimeryk**
Adds a loading placeholder in Email management page (`/domains/manage/email/example.com`).

Before:
screen shot 2016-03-10 at 02 47 18

After:
screen shot 2016-03-10 at 02 44 52

/cc @aidvu @umurkontaci

### CODE REVIEW

File: client/my-sites/upgrades/domain-management/style.scss [hunk 1]
```diff
@@ -949,6 +949,22 @@ ul.dns__list {
 	}
 }
 
+.domain-management-email {
+	.is-placeholder {
```
Comment (mtias) [inline client/my-sites/upgrades/domain-management/style.scss]: Should disable the `::after` on the element that has the fade effect :)

File: client/my-sites/upgrades/domain-management/email/index.jsx [hunk 2]
```diff
@@ -68,7 +69,7 @@ const Email = React.createClass( {
 
 	content() {
 		if ( ! ( this.props.domains.hasLoadedFromServer && this.props.googleAppsUsersLoaded ) ) {
-			return this.translate( 'Loading…' );
+			return this.loadingPlaceholder();
```
Comment (mtias) [inline client/my-sites/upgrades/domain-management/email/index.jsx]: Can we just return `<Placeholder>` here?

File: client/my-sites/upgrades/domain-management/email/google-apps-user-item.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	},
+
+	shouldComponentUpdate( nextProps ) {
+		return this.props.user !== nextProps.user || this.props.onClick !== nextProps.onClick;
+	},
+
+	render() {
+		return (
+			<li>
+				<span className="google-apps-users-card__user-email">
```
Comment (mtias) [inline client/my-sites/upgrades/domain-management/email/google-apps-user-item.jsx]: Do we have an issue for cleaning up these classes?

File: client/my-sites/upgrades/domain-management/email/index.jsx (hunk 2 above)
Comment (klimeryk) [inline client/my-sites/upgrades/domain-management/email/index.jsx]: I was just following the pattern I've noticed here - no actual components in `content()`, everything is delegated to another render function. I know it's an overkill for the loading placeholder and I can change it to inline if I have indeed overdone it :wink:

File: client/my-sites/upgrades/domain-management/email/google-apps-user-item.jsx (hunk 3 above)
Comment (klimeryk) [inline client/my-sites/upgrades/domain-management/email/google-apps-user-item.jsx]: Not that I know of. But just to clarify - what would need to be cleaned up in this case?

File: client/my-sites/upgrades/domain-management/email/index.jsx (hunk 2 above)
Comment (mtias) [inline client/my-sites/upgrades/domain-management/email/index.jsx]: Haha, yes, I didn't notice the indirection. I do think it's a bit overdone :)

File: client/my-sites/upgrades/domain-management/email/google-apps-user-item.jsx (hunk 3 above)
Comment (mtias) [inline client/my-sites/upgrades/domain-management/email/google-apps-user-item.jsx]: The component is `email` which is too generic. `google-apps-user-item.jsx` is a subcomponent, so classes should be prefixed to the component. See https://wpcalypso.wordpress.com/devdocs/docs/components.md

**Comment by klimeryk** [comment]
Thanks, @mtias, for the comments - I've moved "up" `<Placeholder />` and corrected the class names.

**Comment by mtias** [comment]
:+1:

**Pull request merged**